Failing on eiger with paraview/6.1:2422729971

```
CMake Error at /user-environment/paraview/lib64/cmake/paraview/ParaViewPlugin.cmake:1163 (message):
  Failed to find the required module VTK::hdf5vtk.
```

--- a/checks/apps/paraview/build-gadget-plugin/paraview_buildgadgetplugin.py
+++ b/checks/apps/paraview/build-gadget-plugin/paraview_buildgadgetplugin.py
@@ -28,8 +28,8 @@ class ParaviewBuildGadgetPlugin(rfm.CompileOnlyRegressionTest):
             'git clone --depth 1 --branch master https://github.com/jfavre/ParaViewGadgetPlugin gadget-plugin.git',
         ]
 
-        self.build_system.cc = "mpicc"
-        self.build_system.cxx = "mpicxx"
+        self.build_system.cc = "gcc"
+        self.build_system.cxx = "g++"
         self.build_system.builddir = "build"
         self.build_system.configuredir = "gadget-plugin.git"
 

--- a/checks/apps/paraview/build-gadget-plugin/paraview_buildgadgetplugin.py
+++ b/checks/apps/paraview/build-gadget-plugin/paraview_buildgadgetplugin.py
@@ -1,4 +1,4 @@
-# Copyright 2016 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# Copyright ETH Zurich/Swiss National Supercomputing Centre (CSCS)
 # ReFrame Project Developers. See the top-level LICENSE file for details.
 #
 # SPDX-License-Identifier: BSD-3-Clause

--- a/checks/apps/paraview/build-gadget-plugin/paraview_buildgadgetplugin.py
+++ b/checks/apps/paraview/build-gadget-plugin/paraview_buildgadgetplugin.py
@@ -1,0 +1,49 @@
+# Copyright 2016 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class ParaviewBuildGadgetPlugin(rfm.CompileOnlyRegressionTest):
+    valid_systems = ['+uenv']
+    valid_prog_environs = ['+paraview']
+
+    build_system = 'CMake'
+    build_locally = False
+
+    num_tasks = 1
+    num_tasks_per_node = 1
+    time_limit = '3m'
+
+    maintainers = ['jfavre', 'albestro', 'SSA']
+    tags = {'production'}
+
+    @run_before('compile')
+    def prepare_build(self):
+        self.prebuild_cmds = [
+            'git clone --depth 1 --branch master https://github.com/jfavre/ParaViewGadgetPlugin gadget-plugin.git',
+        ]
+
+        self.build_system.cc = "mpicc"
+        self.build_system.cxx = "mpicxx"
+        self.build_system.builddir = "build"
+        self.build_system.configuredir = "gadget-plugin.git"
+
+    @sanity_function
+    def validate_build(self):
+        return sn.all(
+            [
+                sn.assert_eq(
+                    sn.count(sn.glob(r'**/libGadgetReader.so', recursive=True)),
+                    1,
+                ),
+                sn.assert_eq(
+                    sn.count(sn.glob(r'**/pvGadgetReader.so', recursive=True)),
+                    1,
+                ),
+            ]
+        )

